File: Prime-generator/Python/1.py
import sys
import time

def is_prime(n):
    prime = True
    for i in range(2, n - 1):
        if n % i == 0:
            prime = False
    return prime


start = time.time_ns()
primes = []
for i in range(2, int(sys.argv[1])):
    if is_prime(i):
        primes.append(i)
duration = time.time_ns() - start
print(f"Execution time: {duration} ns")

File: Prime-generator/Javascript/1.js
function is_prime(n) {
    let prime = true
    for (let i=2; i<n-1; i++) {
        if (n % i == 0)
            prime = false
    }
    return prime
}

const start = process.hrtime.bigint();
const primes = [];
for (let i=2; i<process.argv[2]; i++) {
    if (is_prime(i)) {
        primes.push(i)
    }
}
const end = process.hrtime.bigint();
const duration = end - start;
console.log(`Execution time: ${duration} ns`);
